test/built-ins/RegExp/escape/escaped-syntax-characters-mixed.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2024 Leo Balter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-regexp.escape
+description: Escaped characters (mixed assertions)
+info: |
+  RegExp.escape ( string )
+
+  This method produces a new string in which certain characters have been escaped.
+  These characters are: . * + ? ^ $ | ( ) [ ] { } \
```

Comment:
Also each of

```
, - = < > # & ! % : ; @ ~ ' ` "
```
per step 3, and `/` per step 1 (the `c is U+002F (SOLIDUS)` case).

Test below asserts that those are not escaped, but that's wrong. They're a different branch for editorial reasons, but they're still escaped just the same as these.